src/reducers/book.js

```diff
@@ -0,0 +1,38 @@
+import { ADD_BOOK, DEL_BOOK } from '../actions/index';
+
+const defaultState = [
```

Comment:
Our book object should be in `src/index.js` right?
![image](https://user-images.githubusercontent.com/31897434/79176379-fd66fa80-7df7-11ea-8218-aeaef411f8bc.png)